### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ivanzh13**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,26 @@
...
+class InvalidCommandError(Exception):
+    ...
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) != 3 or command_list[0] != "mv":
+        raise InvalidCommandError("Unknown command.")
+
+    if "/" in command_list[2]:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:13]: It is bad solution to check by /
Instead use os.path.split()

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1 +1,25 @@
...
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) != 3 or command_list[0] != "mv":
+        raise InvalidCommandError("Unknown command.")
+
+    path = ""
+    for folder in command_list[2].split("/")[:-1]:
+        path = os.path.join(path, folder)
+        if not os.path.isdir(path):
+            os.mkdir(path)
```
Comment (ArsenPidhoretskyi) [inline app/main.py:17]: First of all use os.path.split
And then use os.makedirs instead of loop

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1 +1,22 @@
...
+
+class InvalidCommandError(Exception):
+    ...
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) != 3 or command_list[0] != "mv":
+        raise InvalidCommandError("Unknown command.")
+
```
Comment (vasylhnatiuk) [inline app/main.py:12]: add `command_name, origin_path, new_path = command_list` and use for more readable code.

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 4]
```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+class InvalidCommandError(Exception):
+    ...
+
+
+def move_file(command: str) -> None:
+    command_name, origin_path, new_path = command.split()
```
Comment (vasylhnatiuk) [inline app/main.py:9]: you must do it after 11 line because if len(command.split()) != 3 code will be broken

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 5]
```diff
@@ -1 +1,27 @@
...
+    ...
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+
+    if len(command_list) != 3 or command_list[0] != "mv":
+        raise InvalidCommandError("Unknown command.")
+
+    _, origin_path, new_path = command.split()
```
Comment (vasylhnatiuk) [inline app/main.py:14]:
```suggestion
    _, origin_path, new_path = command_list
```

Comment (vasylhnatiuk) [reply app/main.py:14]: better to reuse data

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by vasylhnatiuk — APPROVED** [review]

**Pull request closed without merging**